client/rpc/util.go

```diff
@@ -0,0 +1,24 @@
+package rpc
+
+import (
+	"fmt"
+
+	"github.com/statechannels/go-nitro/network/serde"
+)
+
+// getTopics returns a list of topics that the client/server should subscribe to.
+func getTopics() []string {
```

Comment:
How central is the idea of "topics" to RPC in general? Is this something peculiar to NATS (which might not make sense if we used a different transport?)

Reply:
In RPC, there are method requests and responses. We should get away from the concept of topics in transport agnostic code.